src/main/java/server/Routes.java

```diff
@@ -0,0 +1,8 @@
+package server;
+
+public class Routes {
+    private Routes() {
+    }
+
+    public static String[] ROUTES = {"/simple_get", "/get_with_body"};
```

Comment:
Please take a look at this: https://restfulapi.net/resource-naming/.

underscores are usually discouraged. I think the rest of the article also covers good information on naming in general!

Reply:
I'd be happy to change them, but they're specified by the acceptance tests!

Reply:
Got it. Let's keep it as is for now and maybe you can bring this up during the IPM as a possible change in a future sprint. Thanks!